lib/src/main/java/org/sonarsource/scanner/lib/internal/util/OsResolver.java

```diff
@@ -75,6 +77,6 @@ private boolean isAlpine() {
    * Operating systems supported by the JRE provisioning.
    */
   public enum OperatingSystem {
-    LINUX, WINDOWS, MACOS, ALPINE
+    LINUX, WINDOWS, MACOS, ALPINE, Z_OS
```

Comment:
```suggestion
    LINUX, WINDOWS, MACOS, ALPINE, ZOS
```
For consistency with `MACOS` maybe?